### DISCUSSION TITLE
How to use this with enterprise single sign on (sso)?

### DISCUSSION THREAD

**Discussion by samanthakem** [hub4j/github-api#1129] [Q&A]
How to use this with enterprise single sign on (sso)?

**Comment by bitwiseman** [comment]
You'll sign in as user on your  enterprise site and create a personal access token.  Then you'll use that access token the same as you outside of SSO. 

https://docs.github.com/en/github/authenticating-to-github/about-authentication-with-saml-single-sign-on

> To use the API or Git on the command line to access protected content in an organization that uses SAML SSO, you will need to use an authorized personal access token over HTTPS or an authorized SSH key.

**Reply by samanthakem** [reply]
Thank you for the quick answer @bitwiseman.

Do you know if there is a way to use this library to connect through SSH?

**Reply by bitwiseman** [reply]
This library interacts with the REST API of GitHub via HTTPS. As far as I know, GitHub does not offer a way to do that via SSH.
